### DISCUSSION TITLE
python file right click does not display "Go to definition"

### DISCUSSION THREAD

**Discussion by wutachiang** [coder/code-server#4348] [Q&A]
Right-click on the python project does not have the "go to" option, and the function cannot be jumped, but right-click on the c++ project has the "go to" option, and the function can be jumped normally.

**Comment by wutachiang** [comment]
![Screenshot from 2021-10-13 17-51-33](https://user-images.githubusercontent.com/13926708/137241102-88c373de-e281-48b9-ae2b-6f8428bd1019.png)

**Reply by 123hicxs** [reply]
老兄，我也发现使用python的时候没有go to definition，我是用Docker里面启动的code server，用本机浏览器登录的，你是否找到了解决办法与问题所在？
![Uploading image.png…]()

**Comment by jsjoeio** [comment]
Hmm...I'm not sure what this could be. We're working on updating VS Code which could fix this? Keep an eye on the next release and let us know if it's still happening after.

**Reply by 123hicxs** [reply]
When I was debugging a Python program, I found that I couldn't go to definition. I started the code server in the Docker container and locally accessed through browser. Do I need some configuration? Or for other reasons?

**Reply by 123hicxs** [reply]
![Uploading image.png…]()

**Reply by code-asher** [reply]
No configuration should be necessary, it should just work as long as the Python extension is installed as far as I know.  I tested on code-server 4.18.0 and it worked for me after I selected an interpreter (`Python: Select Interpreter` from the command palette).

**Reply by 123hicxs** [reply]
I found the problem yesterday, it's due to the Balance plugin, as it's not open source.
